resources/node-red-log

```diff
@@ -1,4 +1,13 @@
 #!/bin/bash
+LINES=25
+while getopts ":n:" opt; do
+  case $opt in
+    n) LINES="$OPTARG"
+    ;;
+    \?) echo "Invalid option -$OPTARG" >&2
```

Comment:
I'd prefer a slight more positive voice, rather than just telling user they are wrong. EG. Usage: node-red-log [-n lines]

Reply:
Good idea, done. Also, I realized it would execute the command with the default 25 lines anyways, so I added an exit in case of an unrecognized parameter.